playbook-website/Gemfile

```diff
@@ -24,9 +24,6 @@ gem "faker"
 gem "will_paginate"
 gem "sentry-ruby"
 gem "sentry-rails"
-gem 'rack-mini-profiler'
```

Comment:
These were causing issues locally when testing react, and i needed to remove them.